components/esp_phy/src/phy_init.c

```diff
@@ -674,7 +674,7 @@ void esp_phy_load_cal_and_init(void)
     }
 #endif
 
-#if CONFIG_ESP_PHY_ENABLE_USB
+#if CONFIG_SERIAL_ESP32_USB
```

Comment:
phy init will only be called when BT or WiFi is enabled. In this case, if ESP32-C3USB port is used, this will automatically call the `phy_bbpll_en_usb`. This won't required another kconfig in Zephyr.

Reply:
If the usb-serial is enabled elsewhere OR by default, this seems to be the correct fix.